### DISCUSSION TITLE
Analytics granularity

### DISCUSSION THREAD

**Discussion by agateau-gg** [Homebrew/discussions#4252] [Everyday usage] [answered]
### Description of issue

I am gathering some metrics on the adoption of an application available from various sources, one of which being a Homebrew tap. There is some interesting information here https://formulae.brew.sh/analytics/, but the granularity is quite coarse: the most precise is 30 days.

Would it be possible to add finer granularity? 1 day would be awesome, but even 7 days would be great.

**Comment by p-linnane** [comment, marked as answer]
The current granularity is by design. We use this information in aggregate only, and the current split serves this purpose well.

**Reply by agateau-gg** [reply]
OK, that's what I suspected. Thanks for the answer.
